{
	"id": "openSUSE-SU-2024:12094-1",
	"modified": "2024-08-13T21:44:48Z",
	"published": "2024-08-13T21:44:48Z",
	"related": [
		"CVE-2022-28346",
		"CVE-2022-28347"
	],
	"summary": "python310-Django-4.0.4-1.1 on GA media",
	"details": "These are all security issues fixed in the python310-Django-4.0.4-1.1 package on the GA media of openSUSE Tumbleweed.",
	"references": [
		{
			"type": "ADVISORY"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2022-28346"
		},
		{
			"type": "WEB",
			"url": "https://www.suse.com/security/cve/CVE-2022-28347"
		}
	]
}
